backtrace-library/src/main/java/backtraceio/library/metrics/EventsResult.java

```diff
@@ -0,0 +1,98 @@
+package backtraceio.library.metrics;
+
+import com.google.gson.annotations.SerializedName;
+
+import backtraceio.library.models.types.BacktraceResultStatus;
+
+/**
+ * Metrics events send method result
+ */
+public class EventsResult {
+
+    /**
+     * Object identifier
+     */
+    @SerializedName("_rxid")
+    @SuppressWarnings({"UnusedDeclaration"})
+    public String rxId;
+
+    /**
+     * Message
+     */
+    public String message;
+
+    /**
+     * Result status eg. server error, ok
+     */
+    public BacktraceResultStatus status = BacktraceResultStatus.Ok;
+
+    /**
+     * Current report
+     */
+    private EventsPayload eventsPayload;
+
+    /**
+     * HTTP response code in case of error
+     */
+    private int statusCode = -1;
+
+    /**
```

Comment:
I think we don't need to have a constructor definition

Reply:
We can remove it, it's not used. Thank you!